commit 5a6f6401fbccb83222c675dfe2f9834447dacdb7 (refs/changes/48/1648/1)
Author: Brian Bennett <brian.bennett@joyent.com>
Date:   2017-03-10T14:50:19-08:00 (2 years, 7 months ago)
    
    MANTA-3190 Reference to deployment checklist in ops guide should be removed

diff --git a/docs/operator-guide/index.md b/docs/operator-guide/index.md
index 9c19e32..a5f1004 100644
--- a/docs/operator-guide/index.md
+++ b/docs/operator-guide/index.md
@@ -824,10 +824,6 @@ anything from COAL to a multi-compute-node deployment.  The general process is:
 8. If desired, set up connectivity to the "ops", "marlin-dashboard", and
    "madtom" zones.  See "Overview of Operating Manta" below for details.
 
-See the [Service Health
-Checklist](https://mo.joyent.com/docs/manta-deployment/master/service-checklist.html)
-for details on how to check if services are healthy.
-
 
 ## Networking configuration
 
